tests/modifiers.edge-split-modifier.test.three-js-test.html

```diff
@@ -510,12 +512,12 @@
 
         import * as THREE from 'three';
 
-        import { OrbitControls } from './js/OrbitControls.js';
-        import { OBJLoader } from './js/OBJLoader.js';
+        import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+        import { OBJLoader } from 'three/examples/jsm/loaders/OBJLoader.js';
         import { EdgeSplitModifier } from '@dioxygen-software/three-js-extra-public';
-        import * as BufferGeometryUtils from './js/BufferGeometryUtils.js';
+        import { BufferGeometryUtils } from '../node_modules/three/examples/jsm/utils/BufferGeometryUtils.js';
 
-        import { GUI } from './js/lil-gui.module.min.js';
+        import { GUI } from 'dat.gui';
```

Comment:
Hum I don't think this is enough at all. Did you try to execute this ? These are different libraries, the idea was to replace lil with dat to have only one GUI lib in all examples. This should involve updating the GUI code to perform in similar way but with the orther lib.